src/Microsoft.AspNet.Mvc.Razor/Diagnostics/IRuntimeCompilationException.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.Framework.Runtime;
+
+namespace Microsoft.AspNet.Diagnostics
```

Comment:
Not sure about the namespace either. Maybe we can go `Microsoft.Framework` here.